src/components/Lead/Lead.module.css

```diff
@@ -126,4 +134,8 @@
     .caption {
         margin-left: 148px;
     }
+
+    .caption {
```

Comment:
This change is irrelevant to the grid section and should be moved to another PR, but you can leave it here if you do not want to create another one. Personally, I do not like it, but we did not have this requirement at the beginning of the project.